tools/check-style.py

```diff
@@ -29,7 +29,7 @@
 DIFF_REF_COLOR = "\033[36m"
 END_COLOR = '\033[0m'
 
-UNCRUSTIFY_VERSION = "0.61"
+UNCRUSTIFY_VERSION = "0.60"
```

Comment:
Long story short, let's try:

- checking style on CI *always* went through whole files, not *diffs*
- false positives, due to that, have been happening since ages now (it won't change with version being .60 solely)
- IMO a commit fixing style should take place to fix that for good, since having diff checks is much harder with the tools we got
- Some people dislike a patching fixing style, so I did just bump the tool's version. That way, we at least have saner approaches to style fixes when people *do* use the tool (lots of bugs from .60 were corrected)
- I second that re-packaging .61 for Ubuntu now is a PITA, but not convinced that reverting this will bring any real good

:-1: